src/controllers/bearing_info_controller.py

```diff
@@ -28,3 +28,9 @@ def patch_bearing_tags(id, tags):
     bearing = facade.update_bearing_tags(id, tags)
     bearing = to_bearing_detail(bearing)
     return bearing, 200, HEADER
+
+def create_bearing(org_name, status):
+    return "added a new bearing"
+
```

Comment:
just a side note, the function name should be the same as defined in the swagger. otherwise, endpoints won't be mapped correctly.